popsup.js

```diff
@@ -0,0 +1,181 @@
+    imageDesktop: './images/Snapshoot desktop.png',
+    githubIcon: './images/github.png',
+    liveIcon: './images/Icon - Export.png',
+    technologiesMobile: ['Ruby', 'CSS', 'JavaScript'],
+    technologiesDesktop: [
+      'CodeKit',
+      'GitHub',
+      'JavaScript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
+    sourceLink: 'https://github.com/Maxwell011',
+    liveLink: 'https://github.com/Maxwell011/My-Portfolio.git',
+  },
+  {
+    id: 6,
+    name: 'Keeping track of hundreds of components',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    imageDesktop: './images/Snapshoot desktop.png',
+    githubIcon: './images/github.png',
+    liveIcon: './images/Icon - Export.png',
+    technologiesMobile: ['Ruby', 'CSS', 'JavaScript'],
+    technologiesDesktop: [
+      'CodeKit',
+      'GitHub',
+      'JavaScript',
+      'Bootstrap',
+      'Terminal',
+      'Codepen',
+    ],
+    sourceLink: 'https://github.com/Maxwell011',
+    liveLink: 'https://github.com/Maxwell011/My-Portfolio.git',
+  },
+];
+
+const myModal = document.getElementById('myModal');
+// Get the <span> element that closes the modal
+
+/* eslint-disable no-unused-vars */
+const closeModal = () => {
+  myModal.style.display = 'none';
+};
+
+// When the user clicks anywhere outside of the modal, close it
+window.onclick = function modal(event) {
+  if (event.target === myModal) {
+    myModal.style.display = 'none';
+  }
+};
+/* eslint-disable no-unused-vars */
+function viewProjectDetails(projectIndex) {
+  const project = data[projectIndex];
+  myModal.innerHTML = `<div class="modal-content">
+<div class="modal-header">
+  <div class="bg-img">
+  <span onclick="closeModal()" id="closeBtn" class="close">&times;</span>
+  <img class="desktop-bg" src="${project.imageDesktop}" />
+  </div>
+</div>
+<div class="modal-body">
+  <div class="title-container">
+    <h3 id="title">${project.name}</h3>
+  </div>
+  <ul class="tags" id="tech">
+  ${project.technologiesDesktop.map((ele) => `<li>${ele}</>`)}
+  </ul>
+  <ul class="tags" id="mobile-ul">
+  ${project.technologiesMobile.map((ele) => `<li>${ele}</>`)}
+  </ul>
+  <p id="description">${project.description}</p>
+  <div class="modal-footer">
```

Comment:
- [ ] On the desktop screen the project description is overflowing from the model, Please fix this issue and make sure that the description is displayed inside the model and your design matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/189147559-a1e601f2-8e13-4617-982a-5e03181ca3dc.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/189147380-0da2a78b-b6dd-4d43-a97e-0c9e3ecf561d.png)